### PR TITLE
Geal/merge main in v5

### PR DESCRIPTION
**Pull request by Geal**

### CODE REVIEW

**Review by divarvel — APPROVED** [review]
Looks good. My comments are not blocking, since all this has been merged to `main` anyway

File: biscuit-capi/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+biscuit-auth = { version = "5.0.0", path = "../biscuit-auth", features = [
+    "datalog-macro",
+    "serde-error",
+] }
+inline-c = { version = "0.1", optional = true }
+rand = "0.8"
+
+[features]
+default = []
+capi = ["inline-c"]
```
[the remark this answers is not in the document]
Comment (Geal) [reply biscuit-capi/Cargo.toml:22]: I think that can be removed. inline-c is only used in the capi tests. I'll remove it in main first then port to v5

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -2,69 +2,55 @@ name: Rust
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
```
[the remark this answers is not in the document]
Comment (Geal) [reply .github/workflows/rust.yml:5]: good point. I think I need to fix this in a PR for main though

**Comment by Geal** [comment]
I'll make a follow up PR to main to address those comments

**Pull request merged**